### DISCUSSION TITLE
disable highlighting query after execution

### DISCUSSION THREAD

**Discussion by match-gabeflores** [dbeaver/dbeaver#18242] [General]
Dbeaver seems to (sometimes?) highlight (select) the query after the query has complete. is there a way to disable this? 

i'll often have a query running for a few seconds and start typing away somewhere down in the editor. when the execution finishes, it interrupts my typing to highlight the query. sometimes, this causes me to delete the sql query!

not sure if that makes sense. 

On dbeaver 22.2.4. using redshift

**Comment by LonwoLonwo** [comment]
Hm... Can you please provide a video of this issue?

**Comment by match-gabeflores** [comment]
example of the annoying re-highlighting

https://imgur.com/9i0T4s6
https://imgur.com/BaJz1YQ
https://imgur.com/NbUNgAb

i think it's because i highlight the query first. but then i start typing elsewhere. dbeaver re-highlights the original query.

Steps:
- highlight query
- use hotkey to execute statement
- start typing elsewhere
- when query finishes, it highlights the original query. (if you're in the middle of typing, it overwrites the highlighted query! have to use ctrl-z to undo)

**Reply by LonwoLonwo** [reply]
Ok, I can reproduce it. You can create a bug ticket for this with all the additional info.

**Reply by match-gabeflores** [reply]
created here: 
https://github.com/dbeaver/dbeaver/issues/18370
